source/fundamentals/builders/indexes.txt

```diff
@@ -15,9 +15,14 @@ Indexes Builders
 Overview 
 --------
 
-This guide shows you how to specify **indexes** using the Java Driver
-:doc:`builders </fundamentals/builders/>`. Index builders provide
-helper methods for the following types of indexes:
+In this guide, we show you how to use **builders** to specify
+**indexes** for your queries.
```

Comment:
nitpick: I think the original intro was better. The focus is on indexes, not generic builders.